devtools/server/actors/replay/module.js

```diff
@@ -469,13 +477,27 @@ getWindow().docShell.chromeEventHandler.addEventListener(
 let gReactDevtoolsInitialized = false;
 
 gNewGlobalHooks.push(dbgWindow => {
+  initRecordReplayGlobals(dbgWindow);
   if (!gReactDevtoolsInitialized) {
     gReactDevtoolsInitialized = true;
     initReactDevtools(dbgWindow, RecordReplayControl);
     initReduxDevtools(dbgWindow, RecordReplayControl);
   }
 });
 
+// Add some global variables to all windows when recording/replaying.
+function initRecordReplayGlobals(dbgWindow) {
+  const window = dbgWindow.unsafeDereference();
+
+  window.wrappedJSObject.__RECORD_REPLAY_PERSISTENT_ID__ = obj => {
```

Comment:
Are these for testing purposes, or do we expect to use them in a followup? I'm fine to land this for now, but since the usage is a bit unclear it definitely makes it harder to give feedback about these. If it's for react devtools I'd potentially prefer to pass them to that explicitly somehow, for example.